.rubocop.yml

```diff
@@ -0,0 +1,15 @@
+Documentation:
```

Comment:
Great. I like rubocop 👍 
But, could you move all the rules below then `AllCops`?
Also, add 
```
Rails:
  Enabled: true
```